float64[33] position
float64 traj_time
bool gravity
bool relative

